config/packages/security.yaml

```diff
@@ -33,9 +33,17 @@ security:
                 lifetime: 604800 # 1 week in seconds
                 path: /
                 httponly: true
+            access_denied_url: /access-denied
             security: true
 
     access_control:
-        - { path: ^/cars, roles: ROLE_USER }
-        - { path: ^/login, roles: PUBLIC_ACCESS }
-        - { path: ^/register, roles: PUBLIC_ACCESS }
+    - { path: ^/login, roles: PUBLIC_ACCESS }
+    - { path: ^/register, roles: PUBLIC_ACCESS }
+    - { path: ^/cars/create, roles: ROLE_VERIFIED }
```

Comment:
Instead of cars/this cars/that ... put - { path: ^/cars/, roles: ROLE_VERIFIED } it should work (you see / )